views/trust-involved.html

```diff
@@ -45,21 +45,53 @@ <h1 class="govuk-heading-xl">{{ pageParams.title }}</h1>
       {% endfor %}
 
       {% set trustees = [] %}
-      {% for trustee in pageData.trustees %}
-        {% set type = pageData.trusteeTypeTitle[trustee.trusteeItemType] %}
+      {% for trustee in pageData.trusteeData %}
+        {% set type = trustee.type %}
+        {% set name = trustee.forename + " " + trustee.surname %}
         {% set trustees = (
           trustees.push({
             key: {
-              text: type
+              text: type  
             },
             value: {
-              text: trustee.name
+              text: name
+            },
+            actions: {
+              items: [SUMMARY_LIST_LINK(
+                'remove',
+                pageData.beneficialOwnerUrlDetach + '/' + trustee.id,
+                [type, name, "detach from trust button"] | join(' '),
+                "detach-trustee-from-trust-button"
+              )]
             }
           }),
           trustees
         ) %}
       {% endfor %}
 
+      {% for trustee in pageData.trusteeData %}
+        {% set type = trustee.type %}
```

Comment:
trustee.type will bring the wrong data from the trust. type is linked to `roleWithinTrustType` on the `GeneralTrustee` model. That type is relating to the type of indiviudal trustee so whether it's settlor, beneficiary, grantor or interested person. 
We are after the actual type of trustee whether it's a individual, legal entity or historical. GeneralTrustee doesn't contain that data. In my PR I have created TrusteeItem which contains the trustee type and the trustee name. 

I've mapped data from the session object to this. You would need to join the first and last name of the individual so it fits in the name field. 

The way the code is currenly implemented, we wouldn't be getting the `trustee type` displaying on the page and it would also stop the legal entity work from displaying. 

I think we need to do something similar for `individual` like i have for `legal entity` in my PR. 

https://github.com/companieshouse/overseas-entities-web/pull/640

Happy to have a call if you want